webapp/src/hooks/hooks.js

```diff
@@ -23,6 +24,20 @@ export default class Hooks {
             this.store.dispatch(openCreateModalWithoutPost(description, contextArgs.channel_id));
             return Promise.resolve({});
         }
+
+        if (message && (message.startsWith('/jira connect') || message === '/jira connect')) {
```

Comment:
redundant

Reply:
I will address this in a separate PR once this and `subscribe` are merged